{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: opps", "trace": "def test_broken():\n>       raise Exception('opps')\nE       Exception: opps\n\ntest_alluredemo1.py:17: Exception"}, "start": 1608790393606, "stop": 1608790393606, "uuid": "e993227c-96ba-4c58-9bfe-8a70fc26ae18", "historyId": "62e3d0efc8226834b589024da8762ee8", "testCaseId": "17acedbab6e0d57f20e780d7d6084ef4", "fullName": "test_alluredemo1#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo1"}, {"name": "host", "value": "rong360deMacBook-Pro-104.local"}, {"name": "thread", "value": "69601-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo1"}]}